fdbclient/admin_client.go

```diff
@@ -75,10 +75,6 @@ type cliAdminClient struct {
 	// custom parameters that should be set.
 	knobs []string
 
-	// Whether the admin client should be able to run operations through the
-	// client library rather than the CLI.
-	useClientLibrary bool
```

Comment:
I think in the past @brownleej added this for doing some tests with DNS, but I don't remember the specifics and I'm happy to remove this code if we don't need it anymore.

Reply:
I don't really know of a good way to test this change, but (at least in theory), we can delete it.

Reply:
I think we can drop this feature since it was only used once for some manual testing.